Java/README.md

```diff
@@ -179,6 +179,7 @@ _add list here_
 - [Factorial Using Big Integer](math/Big_Integer_Factorial.java)
 - [Polynomial Sum & Product using Linked Lists](math/polynomial.java)
 - [Neon number](Neon_number.java)
+- [ASCII_code_of_each_character_of_string](math/ASCII_code_of_each_character_of_string.java)
```

Comment:
- Remove Underscores from `[ASCII_code_of_each_character_of_string]`
- `ASCII_code_of_each_character_of_string.java` make it concise and short